Source code does not compile with older gcc versions.

make command fails with older versions of the compiler where the default C standard is less c99 since the source uses some features of c99 standards. For example, compiling on a fresh Oracle Linux 7.9 system with:

```
gcc --version
gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44.0.3) 
```
make fails with the following errors:

```
 > make
gcc -Wall -g -c -o src/nuke.o src/nuke.c -Wall -g
src/nuke.c: In function ‘nuke’:
src/nuke.c:75:9: error: ‘for’ loop initial declarations are only allowed in C99 mode
         for (int i = 0; i < nreps; i++) {
         ^
src/nuke.c:75:9: note: use option -std=c99 or -std=gnu99 to compile your code
src/nuke.c: In function ‘rand_drv’:
src/nuke.c:160:9: error: ‘for’ loop initial declarations are only allowed in C99 mode
         for (int i = 0; i < bs; i++) {
         ^
make: *** [src/nuke.o] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 CC = gcc
-CFLAGS = -Wall -g
+CFLAGS = -Wall -g -std=gnu11
 
 BIN = nuke
 

